gnucash/import-export/test/gmock-qofquery.h

```diff
@@ -0,0 +1,46 @@
+#ifndef GMOCK_QOFQUERY_H
+#define GMOCK_QOFQUERY_H
+
+#include <gmock/gmock.h>
+
+#include <qofquery.h>
+#include <qofquery-p.h>
+
+extern "C"
+{
+#include <Query.h>
+}
+
+// mock up for QofQuery
+// hint: class QofMockQuery can not be derived from QofQuery, since struct _QofQuery is not public
```

Comment:
Say "Note:" or "N.B." (nota bene, latin for note well) instead of "hint:".

Reply:
ok

Reply:
Done in PR #738